en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -6,6 +6,9 @@ icon: table
 
 # `GridField`
 
+> [!TIP]
+> GridField can only be used with `$list` data sets that are of the type `SS_List` such as `DataList` or `ArrayList`.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.